python/ctsm/site_and_regional/run_neon.py

```diff
@@ -62,14 +62,11 @@
 import time
 import pandas as pd
 
-from standard_script_setup import *
```

Comment:
I think this is really the only line that has to change for the script to work, but it also likely needs to be fixed in black, to prevent it from being moved again.  Not sure if there's another tag in the queue where this fix could come in...?